ingestion/src/metadata/profiler/orm/converter/base.py

```diff
@@ -156,7 +169,7 @@ def ometa_to_sqa_orm(
 
     if not isinstance(orm, DeclarativeMeta):
         raise ValueError("OMeta to ORM did not create a DeclarativeMeta")
-
+    global_orm_registry[table.fullyQualifiedName.root] = orm
```

Comment:
I think we should be cautious with keeping all form objects in memory (from what I understand this does)

Reply:
I agree but could not a find a better solution for now. see [this comment](https://github.com/open-metadata/OpenMetadata/pull/18716/files/94619e24f57a4c87740c1dfb8b8dabf679405af1#diff-22074ecf4c9865187ab56afe336d18334aabdaea68c4abc37314f4cf9b33b7bbR217-R219) and LMK if you have another solution which is more scalable.

I am also not convinced that the current implementation is any better since it initialisez the orm at an early stage in the cycle so it might also be keeping them in memory. Was this explored via a load test?

Reply:
using `sampler.table` within TestSuiteInterface